code/modules/projectiles/guns/energy/special_eguns.dm

```diff
@@ -469,6 +469,7 @@
 	var/obj/item/assembly/signaler/anomaly/flux/core = null
 	var/has_bluespace_crystal = FALSE
 	var/admin_model = FALSE //For the admin gun, prevents crystal shattering, so anyone can use it, and you dont need to carry backup crystals.
+	ratio = 4
```

Comment:
can you move this above the variables defined on this type up to the rest of the inherited vars

Reply:
sure

Reply:
actually that variable was there for testing and not needed, removed.